docs/vectorizer.md

```diff
@@ -62,11 +63,11 @@ Timescale Cloud or on a self-hosted Postgres server.
 
 - Self-hosted Postgres
 
-  Set an environment variable that is the [same as your API key name](./self-hosting-vectorizer.md#install-and-configure-vectorizer-cli). 
+  Set an environment variable that is the [same as your API key name](./vectorizer-worker.md#install-and-configure-vectorizer-worker). 
   For example:
   ```bash
   export OPENAI_API_KEY="Your OpenAI API key"
-  vectorizer
+  pgai vectrorizer worker
```

Comment:
```suggestion
```
What is this call of vectorizer for here? The `vectorizer.md` explains the SQL vectorizer interface. It has nothing to do with the worker and calling it here is not necessary and just confusing to me.